src/components/DiaryWritePage/DiaryForm/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import * as S from './DiaryForm.style';
+
+import type { Emotion } from '@/@types/types';
+import Button from '@/components/common/Button';
+import EmotionIcon from '@/components/DiaryWritePage/EmotionIcon';
+
+const DiaryForm = () => {
+  const emotions: Emotion[] = ['행복', '기쁨', '쏘쏘', '슬픔', '화남'];
+  const [selectedEmotion, setSelectedEmotion] = useState<Emotion | undefined>(undefined);
+
+  const handleEmotionClick = (emotion: Emotion) => {
+    setSelectedEmotion(emotion);
+  };
+
+  return (
+    <S.Form>
+      <S.Container>
+        <p>오늘의 감정은 어때?</p>
+        <S.FlexRow>
+          {emotions.map((emotion, index) => (
+            <S.Button type={'button'} key={index} onClick={() => handleEmotionClick(emotion)}>
```

Comment:
```suggestion
            <S.Button type="button" key={index} onClick={() => handleEmotionClick(emotion)}>
```
key를 index로 사용해도 되지만, emotin 자체의 value가 겹치지 않는 고유한 값이여서, emotion 자체를 key로 사용해도 될 것 같아요.
(그냥 지나가는 의견입니다:))